### DISCUSSION TITLE
BRAINSTORM - List of things for programming to talk about with judges

### DISCUSSION THREAD

**Discussion by tina-hague** [Patribots4738/Crescendo2024#257] [Ideas]
In the pit, it may be useful to have a list of things to talk about with the judges and show-off in case we forget in the heat of the moment. What are some cool things we could show-off? This list will be shared with Patribots Programming.

If you want to add ideas to the tech binder your can add topics or elaborate on others here -> [Tech Binder Stuffs to Add](<https://docs.google.com/document/d/1hucQVwNfSouOQ_nJJaj0nCyEyzfM1sFyzuf15Yx-vzo/edit?usp=sharing>)

This is our current list:
- Simulation and testing in Advantage Scope
- Tuning PID and speeds w/out redeploying
- Scouting Website and backend
- Discretize
- Note detection and _chase_
- Modular autonomous
- Patribox controller remapping
- Helping 3255 and 1622 with simulation
- F-map

**Comment by Hadizah454** [comment]
Note detection in auto maybe?
